web/locales/en/messages.ftl

```diff
@@ -959,4 +962,4 @@ contribution-just-unsure-title = Just Unsure?
 contribution-just-unsure-description = If you come across something that these guidelines don’t cover, please vote according to your best judgement. If you really can’t decide, use the skip button and go on to the next recording.
 
 see-more = <chevron></chevron>See more
-see-less = <chevron></chevron>See less
\ No newline at end of file
+see-less = <chevron></chevron>See less
```

Comment:
This snuck into the commit - I leave it since .editorconfig mandates a final newline